paasta_tools/cli/cmds/validate.py

```diff
@@ -162,6 +162,9 @@ def get_schema(file_type):
 
     :param file_type: what schema type should we validate against
     """
+    if file_type == "eks":
+        # since eks_schema.json is a symlink to kubernetes_schema.json
+        file_type = "kubernetes"
```

Comment:
ohh, I think I linked the eks_schema.json to the full path instead of just "kubernetes_schema.json" and thus wasn't able to open the file. Will correct that